scripts/build_virtualenv.sh

```diff
@@ -149,7 +149,11 @@ if [ ! -e $VENV/inst/.completed.$SCRIPT_VERSION ]; then
     if [ ! -e $VENV/inst/bin/python ]; then
         pushd cpython-2011
         hg checkout dtrace-issue13405_2.7
-        LDFLAGS="-Wl,-rpath=$VENV/inst/lib" ./configure $EXTRA_CONFIG_OPTS --prefix=$VENV/inst --enable-shared
+        if ( python -c 'import sys; print(sys.maxunicode)' | grep -q '65535' ); then
+            LDFLAGS="-Wl,-rpath=$VENV/inst/lib" ./configure $EXTRA_CONFIG_OPTS --prefix=$VENV/inst --enable-shared
+        else
+            LDFLAGS="-Wl,-rpath=$VENV/inst/lib" ./configure $EXTRA_CONFIG_OPTS --prefix=$VENV/inst --enable-shared --enable-unicode=ucs4
```

Comment:
With this syntax the LDFLAGS are only set for this one invocation of ./configure. Also configure doesn't usually take its configuration options from LDFLAGS. And are you sure LDFLAGS is not used anywhere else in the script that would mess things up?